frontend/src/components/Navbar.js

```diff
@@ -1,25 +1,62 @@
+import React, { useState } from 'react';
 import { Link } from "react-router-dom";
+import './Navbar.css'; // Import the CSS file for styling
 
 const Navbar = () => {
+  const [isLoggedIn, setIsLoggedIn] = useState(false); // State for login status
+
+  const handleLogin = () => {
+    setIsLoggedIn(true);
+  };
+
+  const handleLogout = () => {
+    setIsLoggedIn(false);
+  };
+
   return (
-    <nav
-      className="navbar"
-      style={{
-        display: "flex",
-        justifyContent: "space-between",
-        backgroundColor: "lightblue",
-      }}
-    >
-      <h1> UASC </h1>
-      <div className="links" style={{ display: "flex" }}>
-        <Link to="/">Home</Link>
-        <Link to="/about">About</Link>
-        <Link to="/events">Events</Link>
-        <Link to="/contact">Contact</Link>
-        <Link to="/register">Register</Link>
-        <Link to="/login">Login</Link>
-        <Link to="/profile">Profile</Link>
-        <Link to="/booking">Booking</Link>
+    <nav>
+      <div id="zero" style={{ display: "flex" }}>	
+        <div id="one">
+          <img src="https://uasc.co.nz/wp-content/uploads/2021/05/UASC-LOGO-White.png" alt="logo"/>
+        </div>
+        <div id="two">
+          <ul>
+            <li>
+              <a href="/home">Home</a>
+            </li>
+            <li>
+              <a href="#about">About</a>
```

Comment:
```suggestion
              <a href="/about">About</a>
```
The href for the tag should contain a '/' rather than a '#'